.github/CONTRIBUTING.md

```diff
@@ -1,36 +1,64 @@
 
-## Onde contribuir
+## Ecossistema do Querido Diário
 
-O Querido Diário possui vários repositórios de código para você contribuir:
+O Querido Diário possui vários repositórios, possibilitando contribuições onde houver mais afinidade:
 
-- [Documentação e Comunidade](https://github.com/okfn-brasil/querido-diario-comunidade)
 - [Raspadores](https://github.com/okfn-brasil/querido-diario)
-- [Toolbox](https://github.com/okfn-brasil/querido-diario-toolbox)
-- [Infraestrutura](https://github.com/okfn-brasil/querido-diario-infra)
 - [Frontend](https://github.com/okfn-brasil/querido-diario-frontend)
+- [Documentação e Comunidade](https://github.com/okfn-brasil/querido-diario-comunidade)
+- [Toolbox](https://github.com/okfn-brasil/querido-diario-toolbox)
 - [Data Processing](https://github.com/okfn-brasil/querido-diario-data-processing)
 - [API](https://github.com/okfn-brasil/querido-diario-api)
 - [API Wrapper](https://github.com/okfn-brasil/querido-diario-api-wrapper)
+- [Infraestrutura](https://github.com/okfn-brasil/querido-diario-infra)
 - [Censo](https://github.com/okfn-brasil/censo-querido-diario)
 
 ## Como contribuir
 
-Caso você ainda não tenha contribuido com um projeto de código aberto ou não está familiarizado com este projeto, existem tarefas indicadas para pessoas como você na aba [*Issues*](https://github.com/okfn-brasil/querido-diario-comunidade/issues), etiquetadas como [`good first issues`](https://github.com/okfn-brasil/querido-diario-comunidade/issues?q=is%3Aissue+is%3Aopen+label%3A"good+first+issue") (boas primeiras tarefas).
+Conhecer sobre `git` e como a plataforma *GitHub* funciona é um passo fundamental. Indicamos o [tutorial da Escola de Dados](https://escoladedados.org/tutoriais/introducao-ao-git-e-github-colaborando-com-projetos-de-codigo-aberto/) caso você ainda não conheça.
+
+### Descobrindo com o que contribuir
+
+#### Issues
+Na aba **Issues** do repositório escolhido ficam as tarefas pendentes. Veja se uma te interessa.
+- Quando a tarefa é indicada para iniciantes terá uma etiqueta com `good first issue` ("boa primeira tarefa").
+- Ao encontrar uma que interessar, comente que fará a tarefa. Isso poupará outra pessoa de trabalhar em algo que já está sendo resolvido.
+- Já tendo, ou não, escolhido uma *issue* para trabalhar, você pode também comentar dúvidas relevantes relacionadas a tarefa.
+
+#### Milestones
+Se não houver *issues* abertas ou nenhuma te interessar, veja as **Milestones** do repositório de interesse. Elas indicam as metas a serem alcançadas. Explorar os arquivos buscando encontrar espaços de melhorias que ajudem o repositório a atingir uma *milestone* é uma excelente forma de contribuição.
+- Ao encontrar, abra uma *issue* descrevendo a possibilidade de melhoria percebida. Se você já for realizar a tarefa, mencione na mensagem.
+
+#### Discussions
+Caso tenha uma ideia ou sugestão para os rumos do repositório em específico ou do projeto como um todo, crie uma **Discussion** (discussão).
```

Comment:
```suggestion
Caso tenha uma ideia ou sugestão para os rumos do projeto, crie uma **Discussion** (discussão).
```
Ideia ou sugestão para um repositório em específico entra em issues também. Se for compartimentalizado o suficiente é melhor ficar mais próximo do objeto da discussão.